src/current/_includes/releases/v23.2/v23.2.1.md

```diff
@@ -0,0 +1,161 @@
+- The new [`SELECT FOR UPDATE`](https://www.cockroachlabs.com/docs/v23.2/select-for-update) implementation used under [Read Committed isolation](https://www.cockroachlabs.com/docs/v23.2/read-committed) (and under [Serializable isolation](https://www.cockroachlabs.com/docs/v23.2/demo-serializable) when the `optimizer_use_lock_op_for_serializable` [session variable](https://www.cockroachlabs.com/docs/v23.2/session-variables) is `true`) now locks all [column families](https://www.cockroachlabs.com/docs/v23.2/column-families) instead of only the first column family. [#116826][#116826]
+- Fixed a bug where [`SELECT FOR UPDATE`](https://www.cockroachlabs.com/docs/v23.2/select-for-update) under [Read Committed isolation](https://www.cockroachlabs.com/docs/v23.2/read-committed) on multi-column-family tables was not locking [column families](https://www.cockroachlabs.com/docs/v23.2/column-families) containing only key columns. [#116826][#116826]
+- It is now possible to run `CALL` statements with [`EXPLAIN`](https://www.cockroachlabs.com/docs/v23.2/explain). The `EXPLAIN (OPT)` variant will show the body of the procedure, while other variants will only show the procedure name and arguments. [#116274][#116274]
+- [`EXPLAIN`](https://www.cockroachlabs.com/docs/v23.2/explain) output now contains detailed information about the plans for `CASCADE` actions. [#117719][#117719]
+
+<h3 id="v23-2-1-operational-changes">Operational changes</h3>
+
+- Per-node [hot ranges](https://www.cockroachlabs.com/docs/v23.2/ui-hot-ranges-page) logging now logs the top 5 hot ranges on the local node instead of the top 5 hot ranges cluster-wide. [cockroachdb/cockroach#118334][#118334]
+
+<h3 id="v23-2-1-command-line-changes">Command-line changes</h3>
+
+- Added a new command `cockroach debug enterprise-check-fips`, which diagnoses errors in [FIPS](https://www.cockroachlabs.com/docs/v23.2/fips) deployments. [#115202][#115202]
+- The new flag `--enterprise-require-fips-ready` can be added to any [`cockroach` command](https://www.cockroachlabs.com/docs/v23.2/cockroach-commands) to prevent startup if certain prerequisites for [FIPS](https://www.cockroachlabs.com/docs/v23.2/fips) compliance are not met. [#115202][#115202]
+- [`cockroach workload`](https://www.cockroachlabs.com/docs/v23.2/cockroach-workload) commands now appropriately invoke `.Close` in the case of an error. [#116487][#116487]
+
+<h3 id="v23-2-1-db-console-changes">DB Console changes</h3>
+
+- Updated the "CPU Time" label on the [Runtime Dashboard](https://www.cockroachlabs.com/docs/v23.2/ui-runtime-dashboard) to "SQL CPU Time" and added clarifications to its tooltip. [#116449][#116449]
+- [Statement bundles](https://www.cockroachlabs.com/docs/v23.2/ui-statements-page#diagnostics) are now enabled for Serverless clusters. [cockroachdb/cockroach#117529][#117529]
```

Comment:
ordinarily I'd agree.  in this case I'd rather not since this is a release note. because at this static moment in time when v23.2.1 is released, there is a thing called "serverless".  And I'd rather not have the name get rewritten if/when we change naming and strategy again because that new name won't necessarily mean anything in this (now past) context

Reply:
super good point